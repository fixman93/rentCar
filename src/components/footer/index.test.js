import React from 'react'
import Adapter from 'enzyme-adapter-react-16';

import { shallow, configure } from 'enzyme'

import { Footer } from './index'

configure({ adapter: new Adapter() });

describe('Footer', () => {
  const footer = shallow(<Footer />)
  it('render footer', () => {
    expect(footer).toMatchSnapshot()
  })


})