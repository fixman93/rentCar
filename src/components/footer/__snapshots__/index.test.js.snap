// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render footer 1`] = `ShallowWrapper {}`;
